### PR TITLE
Fix formatting and compilation

### PR DESCRIPTION
**Pull request by arichiv**
Let's make the formatting consistent and resolve any outstanding compiler issues/warnings.

***
<a href="https://pr-preview.s3.amazonaws.com/WICG/client-hints-infrastructure/pull/97.html" title="Last updated on Feb 2, 2022, 12:01 AM UTC (44a68b2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/client-hints-infrastructure/97/fb0a799...44a68b2.html" title="Last updated on Feb 2, 2022, 12:01 AM UTC (44a68b2)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -112,59 +113,68 @@ Accept-CH cache {#accept-ch-cache-definition}
...
-11. For each |feature|->|allowlist| of |policyDirective|:
-    1. If |feature| is not a [=client hints token=], then continue.
-    2. If |allowlist| is the special value `*`, then continue.
-    3. [=set/Append=] |feature| to |settingsObject|'s [=environment settings object/client hints set=] and [=set/Append=] |allowList| to |permissionsPolicy[feature]|'s [[PERMISSIONS-POLICY|permissions policy]].
+<ol>
+ <li>Let |metaElement| be the <{meta}> element.
+ <li>If |metaElement| is not a child of a <{head}> element, then return.
+ <li>If |metaElement| has no <{meta/content}> attribute, or if that attribute's value is the empty string, then return.
+ <li>If |metaElement| has any <a href="https://dom.spec.whatwg.org/#concept-tree-preceding">preceding</a> <a href="https://dom.spec.whatwg.org/#concept-tree-sibling">sibling</a> <{link}> or <{script}> elements (or if any <{link}> or <{script}> elements have begun to execute), then return.
+ <li>Let |acceptCHValue| be the value of |metaElement|'s <{meta/content}> attreibute.
```
Comment (miketaylr) [inline index.bs:164]: nit: typo in "attreibute"

Comment (miketaylr) [reply index.bs:164]: Also, is this step new? Was |acceptCHValue| down on the old step 10 previously undefined?

**Review by miketaylr — APPROVED** [review]
LGTM w/ nits

**Comment by miketaylr** [comment]
Also, please squash all the "update index.bs" commits before merging - either manually or via the "Squash and merge" button.

File: index.bs (hunk 1 above)
Comment (arichiv) [reply index.bs]: Done, and yeah it’s new because it somehow got omitted before.

**Pull request merged**